File: OneToOneCallSample/app/src/main/java/com/stringee/apptoappcallsample/stringee/common/CallConstants.java
package com.stringee.apptoappcallsample.stringee.common;

/** Shared intent actions, extras, notification IDs, and timeout values for the call flow. */
public final class CallConstants {
    public static final String TAG = "StringeeSample";

    public static final String ACTION_INCOMING_PUSH =
            "com.stringee.onetoonecallsample.action.INCOMING_PUSH";
    public static final String ACTION_INCOMING_SDK =
            "com.stringee.onetoonecallsample.action.INCOMING_SDK";
    public static final String ACTION_HIDE_INCOMING =
            "com.stringee.onetoonecallsample.action.HIDE_INCOMING";
    public static final String ACTION_START_IN_CALL =
            "com.stringee.onetoonecallsample.action.START_IN_CALL";
    public static final String ACTION_REJECT =
            "com.stringee.onetoonecallsample.action.REJECT";
    public static final String ACTION_HANG_UP =
            "com.stringee.onetoonecallsample.action.HANG_UP";
    public static final String ACTION_MEDIA_DISMISSED =
            "com.stringee.onetoonecallsample.action.MEDIA_DISMISSED";

    public static final String EXTRA_CALL_ID = "stringee_call_id";
    public static final String EXTRA_CALL_GENERATION = "stringee_call_generation";
    public static final String EXTRA_SESSION_GENERATION = "stringee_session_generation";
    public static final String EXTRA_PENDING_ACTION = "stringee_pending_action";
    public static final String EXTRA_CALLER = "stringee_caller";
    public static final String EXTRA_CALLER_ALIAS = "stringee_caller_alias";
    public static final String EXTRA_VIDEO = "stringee_video";
    public static final String EXTRA_ENGINE = "stringee_engine";
    public static final String EXTRA_ANSWER = "stringee_answer";
    public static final String EXTRA_STARTED_AT = "stringee_started_at";

    public static final int INCOMING_NOTIFICATION_ID = 28011996;
    public static final int ONGOING_NOTIFICATION_ID = 28011997;
    public static final int MEDIA_NOTIFICATION_ID = 14101997;

    private CallConstants() {
    }
}

File: Kotlin-OneToOneCallSample/app/src/main/java/com/stringee/kotlin_onetoonecallsample/stringee/common/CallConstants.kt
package com.stringee.kotlin_onetoonecallsample.stringee.common

/** Shared intent actions, extras, notification IDs, and timeout values for the call flow. */
object CallConstants {
    const val TAG: String = "StringeeSample"

    const val ACTION_INCOMING_PUSH: String = "com.stringee.onetoonecallsample.action.INCOMING_PUSH"
    const val ACTION_INCOMING_SDK: String = "com.stringee.onetoonecallsample.action.INCOMING_SDK"
    const val ACTION_HIDE_INCOMING: String = "com.stringee.onetoonecallsample.action.HIDE_INCOMING"
    const val ACTION_START_IN_CALL: String = "com.stringee.onetoonecallsample.action.START_IN_CALL"
    const val ACTION_REJECT: String = "com.stringee.onetoonecallsample.action.REJECT"
    const val ACTION_HANG_UP: String = "com.stringee.onetoonecallsample.action.HANG_UP"
    const val ACTION_MEDIA_DISMISSED: String =
        "com.stringee.onetoonecallsample.action.MEDIA_DISMISSED"

    const val EXTRA_CALL_ID: String = "stringee_call_id"
    const val EXTRA_CALL_GENERATION: String = "stringee_call_generation"
    const val EXTRA_SESSION_GENERATION: String = "stringee_session_generation"
    const val EXTRA_PENDING_ACTION: String = "stringee_pending_action"
    const val EXTRA_CALLER: String = "stringee_caller"
    const val EXTRA_CALLER_ALIAS: String = "stringee_caller_alias"
    const val EXTRA_VIDEO: String = "stringee_video"
    const val EXTRA_ENGINE: String = "stringee_engine"
    const val EXTRA_ANSWER: String = "stringee_answer"
    const val EXTRA_STARTED_AT: String = "stringee_started_at"

    const val INCOMING_NOTIFICATION_ID: Int = 28011996
    const val ONGOING_NOTIFICATION_ID: Int = 28011997
    const val MEDIA_NOTIFICATION_ID: Int = 14101997
}
